(Actions) update Events page in App Builder (#34506)
DataDog/documentation: content/en/actions/app_builder/events.md

Reviewer: This can be separate, but we should take a pass through all the docs to change `query` --> `action`

   1. Click the **Build with AI** icon (**<i class="icon-bits-ai"></i>**).
   1. Enter a custom prompt for an event, or try the prompt `How can you help me with events?`.

### Events and reactions

In App Builder, events and reactions work together to determine component behavior. 

> Events are triggered both by components (such as a button click or a form submit) and actions (execution finished). Events differ depending on which component you use. For component-specific event details, see [Components][1].

Reviewer 2: ```suggestion
Components (such as button clicks or form submissions) and actions (such as execution finished) can both trigger events. Events differ depending on which component you use. For component-specific event details, see [Components][1].
```
"Execution finished" is the name of an action? It sounds like a notification that an action is complete? This sounds a little awkward as a result.
